tensorflow/core/util/rocm_kernel_helper.h

```diff
@@ -0,0 +1,21 @@
+#if TENSORFLOW_USE_ROCM
```

Comment:
Please kill this file. There is another gpu_kernel_helper which does the same thing